versions/3.1.0.md

```diff
@@ -91,7 +91,7 @@ Some examples of possible media type definitions:
   application/vnd.github.v3.patch
 ```
 ##### <a name="httpCodes"></a>HTTP Status Codes
-The HTTP Status Codes are used to indicate the status of the executed operation. 
+The HTTP Status Codes are used to indicate the status of the executed operation.
```

Comment:
@philsturgeon could you change your editor settings to preserve whitespace at the end of lines in markdown files? (In some markdown variants two trailing spaces equals a hard line break, though I don't know if we depend on that anywhere). It makes it difficult to see what you've actually changed. Also affects #1946

Reply:
@MikeRalphson in the meantime, on the diff page under the settings menu (the gear thing) there is an "Ignore Whitespace" option that you can turn on which takes out the visual noise.